resources/test.json

```diff
@@ -54,6 +54,20 @@
             "test": "paratest --version",
             "tags": ["test"]
         },
+        {
+            "name": "pest",
+            "summary": "an elegant PHP testing Framework with a focus on simplicity, meticulously designed to bring back the joy of testing in PHP",
```

Comment:
Description is a bit long. Can you shorten it to "an elegant PHP testing Framework with a focus on simplicity" ?